user-guide/Advanced_Modules/DOM/tutorials/DOM_Getting_Started_With_DOM.md

```diff
@@ -0,0 +1,270 @@
+1. Click the publish icon in the top right corner of the app.
+
+   You have now published a first version of the app, so you can take a look at what you have created so far.
+
+   ![DOM LCA Publish app](~/user-guide/images/DOM_Getting_Started_with_LCA_PublishApp.png)
+
+1. In your low-code app, fill out the form and save it.
+
+   At this point, you cannot see the created event records yet. For this, you need to add a table listing all event records in the app. This table will use a GQI query.
+
+1. Go back to edit mode, start editing the page again, and create a query:
+
+   1. Expand the *Queries* section in the data pane.
+
+   1. Click the + icon to create a new query.
+
+   1. Enter `Events` as the name of the query.
+
+   1. In the *Select data source* box, select *Get object manager instances*.
+
+   1. Select the module *eventmanagement*, and the definition *Event*.
+
+   1. In the *Select operator* box, select *Select*.
+
+      This will allow you to pick which fields of the event you want to list in the app.
+
+   1. Select the *Event Name*, *Start*, *End*, and *Notes* fields.
+
+   ![DOM LCA Get Dom Instances](~/user-guide/images/DOM_Getting_Started_with_DOM_GetDomInstances.png)
+
+1. Drag and drop the newly created *Events* query on to the page next to the form to add a component using this data.
+
+1. In the component, select *Table* as the visualization.
+
+   ![DOM LCA Events Table Visualization](~/user-guide/images/DOM_Getting_Started_with_DOM_LCA_EventsTableVisualization.png)
+
+1. Click the publish icon in the top right corner of the app to publish the app again.
+
+   The app should now show a table containing the event record you created earlier.
+
+   ![DOM LCA Events Table](~/user-guide/images/DOM_Getting_Started_with_LCA_EventsTable.png)
+
+   At this point, if you add more records through the form, you will not immediately see the records showing up in the table. You will need to refresh the page manually first. To make the even records show up immediately as soon as they have been entered, another event has to be added to the page.
+
+1. Go back to edit mode and add an event to the page to refresh after the *Save* action:
+
+   1. In the pane on the left, under *Header bar* > *Events*, click the icon next to *On click*.
+
+      This will open the dialog with the save action you configured earlier.
+
+   1. Click *Upon completion*.
+
+      ![DOM LCA DON Form Upon Completion](~/user-guide/images/DOM_Getting_Started_with_DOM_FormUponCompletion.png)
+
+   1. select *Execute component action* in the first box, *Fetch the data* in the second box, and *Table 2* in the third box.
+
+      ![DOM LCA Save On Click Event](~/user-guide/images/DOM_Getting_Started_with_DOM_LCA_TableFetchTheData.png)
+
+   1. Click *OK*.
+
+1. Publish the app again and try out creating new events again.
+
+You should now have a fully functioning app where users can add records that will show up in the table as soon as they click the *Save* button.
```

Comment:
Could you also add a screenshot of the final app page, so the users get an idea of what this should look like?

Reply:
Done